src/main/java/org/openstreetmap/atlas/geography/atlas/items/complex/buildings/ComplexBuilding.java

```diff
@@ -58,6 +59,34 @@ protected ComplexBuilding(final AtlasEntity source)
         }
     }
 
+    /**
+     * @return The building's base height
+     */
+    public Optional<Distance> baseHeight()
+    {
+        Map<String, String> tags = getSource().getTags();
+        String minHeightTag = tags.get(MinHeightTag.KEY);
+        try
+        {
+            if (minHeightTag != null)
+            {
+                return Optional.of(HEIGHT_CONVERTER.convert(minHeightTag));
+            }
+            tags = this.outlineSource.getTags();
```

Comment:
Consider using `Validators.hasValuesFor(...)` here instead of grabbing allTags - this can be somewhat expensive.